derp/derphttp/websocket.go

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
-//go:build linux || js
+//go:build linux || windows || darwin || js
```

Comment:
Would prefer to still exclude this on iOS (by using `(darwin && !ios)`), where we're still somewhat memory constrained.